xlive/H2MOD.cpp

```diff
@@ -1118,8 +1119,8 @@ void H2MOD::ApplyHooks() {
 		pspawn_player = (spawn_player)DetourFunc((BYTE*)this->GetBase() + 0x55952, (BYTE*)OnPlayerSpawn, 6);
 		VirtualProtect(pspawn_player, 4, PAGE_EXECUTE_READWRITE, &dwBack);
 
-		PatchCall(GetBase() + 0x86C3, onGameEngineChange);
-		p_originalFunc = (originalFunc)(GetBase() + 0x48CA5);
+		PatchCall(GetBase() + 0x49E95, onGameEngineChange);
```

Comment:
why does this keep changing?

Reply:
why did you also change the original code I had that did this

Reply:
Well, the first hook i did was causing crashes if player was restarting the mission, and the second code change was causing AI not to spawn in MP, because all the AI initialization was done before this method, so now I change it again so both issues are gone. I promise this is the last change I do to this.